ui/src/app/views/workspace-edit.tsx

```diff
@@ -390,8 +434,45 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
         cloneUserRole: false,
         loading: false,
         showUnderservedPopulationDetails: false,
-        showStigmatizationDetails: false
+        showStigmatizationDetails: false,
+        autocompleteLoading: false,
+        autocompleteDiseases: [],
+        diseaseHover: [],
+        dropDown: false,
+        searchTerm: '',
+        hovering: false
       };
+      this.searchTermChangedEvent = fp.debounce(300, this.diseaseSearch);
+    }
+
+    get showSearchResults(): boolean {
+      return !this.state.autocompleteLoading &&
+        this.state.autocompleteDiseases.length > 0 &&
+        !isBlank(this.state.searchTerm) && this.state.dropDown;
+    }
+
+    diseaseSearch(value: string): void {
+      this.setState({autocompleteLoading: true, autocompleteDiseases: [], searchTerm: value});
+      if (!value.trim()) {
+        this.setState({autocompleteLoading: false, dropDown: false});
+        return;
+      }
+      const self = this;
+      const url = 'https://firecloud-orchestration.dsde-dev.broadinstitute.org' +
```

Comment:
This will not work for all environments since its hard coded to the dev environment. We should be able to use the generated client for this which will take care of the request configuration.

Reply:
I don't know if we have a generated firecloud client, but if you want to use fetch, we should at least pull down the FC base URL from the config. Otherwise could go through the firecloud generated client. https://github.com/all-of-us/workbench/blob/master/api/config/config_local.json#L4

Reply:
This PR: https://github.com/all-of-us/workbench/pull/2262/files is an example of how to pull down server config information to the UI.

Reply:
Spoke with Eric and we agreed that using the config would be the faster choice for now, but created https://precisionmedicineinitiative.atlassian.net/browse/RW-3057 to use the generated client in the future.